Doppler.Sap/Models/DopplerUserDTO.cs

```diff
@@ -11,7 +11,7 @@ public class DopplerUserDto
         public string FirstName { get; set; }
         public string LastName { get; set; }
         public string Company { get; set; }
-        public int? StateId { get; set; }
+        public string StateId { get; set; }
```

Comment:
I think that this property is not used, can you remove it?

Reply:
Done